spec/helpers/field_validation_helper_spec.rb

```diff
@@ -127,4 +127,17 @@
       expect(invalid_fields).to eq [{ field: "email_address", text: I18n.t("coronavirus_form.errors.email_format") }]
     end
   end
+
+  describe "#email_address_suggestion" do
+    let(:valid_email_address) { "me@example.com" }
+    let(:invalid_email_address) { "me@example..com" }
```

Comment:
Would be good get a few more examples in here e.g.

```ruby
    %w(
      govuk-coronavirus-services@digital.cabinet_office,gov.uk
      govuk-coronavirus-services@digital
      govuk-coronavirus-services@
      @gov.uk
      me@example..com
      a@b
    ).each do |invalid_email|
      it "does not move to next step with an invalid email address of #{invalid_email}" do
        expect(email_address_suggestion(invalid_email_address)).to eq(valid_email_address)
      end
    end
```

Reply:
Isn't that just testing the gem though?